src/lightning_lite/plugins/precision/__init__.py

```diff
@@ -11,15 +11,17 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from lightning_lite.plugins.precision.precision import Precision  # isort:skip
```

Comment:
Had to move this to the top, otherwise circular import at import of double

Reply:
I don't get why it would be circular. What is imported in Double precision that would create the circle? If we just import both of them here it shouldn't create a circle?

Reply:
If the import is like this in the init file:

(1) from lightning_lite.plugins.precision.other import OtherClassThatSubclassesPrecision
(2) from lightning_lite.plugins.precision.precision import Precision
 
Then while loading the init, it first goes into (1), but other.py tries to import Precision so it goes back to this file and tries to rerun (1), and then you get "cant import blabla due to partially initialized module"